operator/webhooks/pod_mutating_webhook.go

```diff
@@ -60,6 +61,18 @@ func (a *PodMutatingWebhook) Handle(ctx context.Context, req admission.Request)
 		return admission.Errored(http.StatusBadRequest, err)
 	}
 
+	// check if Lifecycle Controller is enabled for this namespace
+	namespace := &corev1.Namespace{}
+	if err = a.Client.Get(ctx, types.NamespacedName{Name: req.Namespace}, namespace); err != nil {
+		logger.Error(err, "could not get namespace", "namespace", req.Namespace)
+		return admission.Errored(http.StatusInternalServerError, err)
+	}
+
+	if namespace.GetAnnotations()[common.NamespaceEnabledAnnotation] != "enabled" {
+		logger.Info("namespace is not enabled for lifecycle controller", "namespace", req.Namespace)
+		return admission.Allowed("namespace is not enabled for lifecycle controller")
```

Comment:
```suggestion
		return admission.Denied("namespace is not enabled for lifecycle controller")
```

Reply:
why should we deny admission, when a namespace is not enabled for the lifecycle controller? In this case, we have to allow the admission.

Reply:
I believe `Allowed` is correct in this case, as the annotation for enabling the KLC on a namespace not being present should not prevent anything from being deployed in that namespace